src/main/java/com/google/sps/data/ArticleBuilder.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.data;
+
+public class ArticleBuilder {
```

Comment:
It's better practice to have the builder inside of the parent class.

Reply:
How do you access the builder if it's inside of the parent class?

Reply:
As Juan mentioned in the daily sync earlier, you can basically copy-paste this whole thing into Article.java. Since it's a public class within a public class, you can simply do something like "Article.Builder builder = new Article.Builder(//params here)" to use it.